packages/runtime/plugin-runtime/src/router/runtime/plugin.tsx

```diff
@@ -43,6 +44,8 @@ export const routerPlugin = ({
     serverBase.find(baseUrl => pathname.search(baseUrl) === 0) || '/';
   let routes: RouteObject[] = [];
   finalRouteConfig = routesConfig;
+  window._SERVER_DATA = parsedJSONFromElement('__MODERN_SERVER_DATA__');
```

Comment:
Should we add an `inBrowser` judgement here?

Reply:
No, have already been distinguished by the `.node.ts` file here